### PR TITLE
export screenbuffer for #37

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

File: src/display/renderloop.jl [hunk 1]
```diff
@@ -281,14 +281,17 @@ function doubleclick(mouseclick, threshold)
...
+screenshot(window; path="screenshot.png", channel=:color) =
+   save(path, screenbuffer(window, channel=channel), true)
+
+function screenbuffer(window; channel=:color)
     fb = window.inputs[:framebuffer]
     channels = fieldnames(fb)[2:end]
     if channel in channels
         img = gpu_data(fb.(channel))[window.area.value]
-        save(path, rotl90(img), true)
+        return rotl90(img)
```
Comment (kleinash) [inline src/display/renderloop.jl]: Somehow this doen't return the rotated image. So the screenshot saved is rotated 90deg on its side.

Comment (SimonDanisch) [inline src/display/renderloop.jl]: Oh totally forget about this... What exactly is the problem? Can you give a reproducible example? (it worked for me like expected so far)

Comment (kleinash) [inline src/display/renderloop.jl]: The image was coming out lying on its side. So I made two separate functions, not that it should have helped but it did. Now I stashed my changes, updated with this copy and its working fine :+1:

**Comment by SimonDanisch** [comment]
overshadowed by JuliaGL/GLWindow.jl#25

**Pull request closed without merging**